### DISCUSSION TITLE
Does it support 2D Rendering?

### DISCUSSION THREAD

**Discussion by MaansenV** [Dreaming381/Latios-Framework#55] [Q&A]
Hello just found this gem and like to ask if it does support 2D, if not is it planned?

**Comment by Dreaming381** [comment]
2D is not supported currently. I don't plan to do the work of supporting it myself, but I am more than happy to provide guidance for anyone who wants to try and add it. If you are interested, this is the starting point: https://github.com/Dreaming381/Latios-Framework/issues/46
